api-controllers/ServiceFabrikApiController.js

```diff
@@ -305,18 +305,23 @@ class ServiceFabrikApiController extends FabrikBaseController {
             resourceType: CONST.APISERVER.RESOURCE_TYPES.DIRECTOR,
             resourceId: req.params.instance_id
           })
-          .catch(() => eventmesh.apiServerClient.createResource({
-            resourceGroup: CONST.APISERVER.RESOURCE_GROUPS.DEPLOYMENT,
-            resourceType: CONST.APISERVER.RESOURCE_TYPES.DIRECTOR,
-            resourceId: req.params.instance_id,
-            parentResourceId: req.params.instance_id,
-            options: {},
-            status: {
-              state: CONST.APISERVER.RESOURCE_STATE.IN_QUEUE,
-              lastOperation: {},
-              response: {}
-            }
-          }))
+          /* jshint unused:false */
+          .catch((NotFound) => {
```

Comment:
Isn't this syntax wrong? I am assuming you plan to use bluebird filtered catch, which should be like:

`.catch(NotFound, e=>{ ... })`

If you're not planning to use filtered catch, then the error variable must be standardized like `notFoundError`